src/components/Firestore/CollectionFilter/ConditionEntry.tsx

```diff
@@ -33,7 +33,7 @@ function getConditionEntryType(value: any) {
 
 export const ConditionEntry: React.FC<{
   name: string;
-  error?: string;
+  error: string | undefined;
```

Comment:
is this logically equivalent change needed? if so, because it's unusual, add a comment. My instinct would be to change it back if I saw it in passing.

Reply:
If you want to require the `error` field to always be specified, but may be `undefined`, please add a comment. (BTW, this is totally acceptable and reasonable since it reduces human mistakes of forgetting to pass `error` through.)